app/javascript/stylesheets/dashboard/dashboard.scss

```diff
@@ -257,6 +255,7 @@
         .empty-card-view{
             grid-column: 1 / span 3;
             text-align: center;
+            background: $background-color;
```

Comment:
We can remove these lines if we remove the background color of the dashboard__card-view class